### PR TITLE
Added a tutorial for nft/listTokens API

### PR DESCRIPTION
**Pull request by aditya-passbird**
https://metakeep-dev.readme.io/reference/create-a-portfolio-of-nfts-you-own

### CODE REVIEW

File: nfts/frontend/src/App.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ function App() {
         <Navbar />
         <Routes>
           <Route path="/" element={<TransferNFT sdk={sdk} />} />
+          <Route path="/nftList" element={<ListNFT sdk={sdk} />} />
```
Comment (rohan-passbird) [inline nfts/frontend/src/App.js:18]: by default let's open the List nft page

Comment (aditya-passbird) [reply nfts/frontend/src/App.js]: Done

Comment (rohan-passbird) [reply nfts/frontend/src/App.js]: Let's delete the transfer page. we don;t need it

**Review by rohan-passbird — APPROVED** [review]
👍

**Pull request merged**